eng/pipelines/templates/stages/archetype-spec-gen-sdk.yml

```diff
@@ -203,6 +193,21 @@ extends:
                 $optional_params
             displayName: 'Generate SDK'
 
+          - task: Powershell@2
```

Comment:
This step should be run only for PR trigger scenarios too.

Reply:
Actually, that can just run in all case because it generates some of the Artifact published